venv/pysnes/cpu.py

```diff
@@ -1792,6 +1865,55 @@ def compute_logic_operation(self, result):
         self.compute_NZflags(flag_result, self.isM())
         return result
 
+    def compute_bit_flags(self, value):
```

Comment:
Why a new flag computation function?

Reply:
Bit uses the data to compute the n and v flag and the result of the accumulator & the data to compute the z flag. compute_NZflags uses one value to compute both the n and the z flag. I could use it to set one and then "overwrite" the other one, but I thought it was easier to make a new function.